ProcessLib/SmallDeformation/SmallDeformationProcessData.h

```diff
@@ -47,6 +56,12 @@ struct SmallDeformationProcessData
 
     std::unique_ptr<MaterialLib::Solids::MechanicsBase<DisplacementDim>>
         material;
+    /// Solid's density. A scalar quantity, Parameter<double>.
```

Comment:
No need to repeat the type, I guess.

Reply:
But it makes a nice link in the docu ;)
https://doxygen.opengeosys.org/ogs_file_param__prj__processes__process__HYDRO_MECHANICS__solid_density.html

Reply:
Touché, a bit at least.